### DISCUSSION TITLE
Add microphone-slash icon

### DISCUSSION THREAD

**Discussion by janhesters** [tailwindlabs/heroicons#807] [Ideas]
There is a `video-camera` icon with a corresponding `video-camera-slash` icon, but there is no `microphone-slash` icon for the `microphone` icon.

A `microphone-slash` icon would be helpful to indicate a muted microphone for real time video apps.

**Comment by trebabcock** [comment]
I second this. I was honestly surprised that it wasn't already a thing.

**Comment by ipetinate** [comment]
@janhesters Hey bro, how are you? I had the same need for this icon, and I ended up creating it in Figma, and I suggested it in this discussion at this link ([https://github.com/tailwindlabs/heroicons/discussions/868](https://github.com/tailwindlabs/heroicons/discussions/868)). If you need to, use the SVG I created, and when they add it to the library, you switch.

**Comment by ndri** [comment]
I added `microphone-slash` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=microphone-slash
